src/compile/legend/rules.ts

```diff
@@ -21,7 +21,7 @@ export function values(legend: Legend) {
   if (vals && isDateTime(vals[0])) {
     return (vals as DateTime[]).map((dt) => {
       // normalize = true as end user won't put 0 = January
-      return timestamp(dt, true);
+      return {signal: dateTimeExpr(dt, true)};
```

Comment:
since it's straightforward how to change this later -- let's make this PR change scale so we can merge and make another PR that does the axis/legend change (so we can merge later when Vega properly supports this.)